### PR TITLE
[FLINK-15991][doc][zh] Translate memory configuration docs into Chinese.

### PR DESCRIPTION
**Pull request by xintongsong**
## What is the purpose of the change

Translate memory configuration docs into Chinese.

## Brief change log

- ede38f8f7c14e35064766f60b5ab9a9db2ea3139: Hotfix for typos and broken links in original English docs.
- 9f09471bfcb67f0fc99934303af634c9910302f2: Translate "Memory Configuration" index.
- 43af8c878946ba402ff939eec8ff67e254c60310: Translate "Set up Task Executor Memory" page
- e5aa5a312a166b6986d475bb2cb490f59d340211: Translate "Detailed Memory Model" page
- 9c7234877fca616ab475bb3d0c2d7ca52448061c: Translate "Memory tuning guide" page
- 647ee6386272af232058070a864909f6689cee75: Translate "Troubleshooting" page
- 44c238715de2eb32237f37d3c38aa42f6765fd6c: Translate "Migration Guide" page

### CODE REVIEW

File: docs/ops/memory/mem_tuning.zh.md [hunk 1]
```diff
@@ -22,66 +22,55 @@ specific language governing permissions and limitations
...
-will dictate the optimal memory configurations of your cluster.
+在部署 Flink 流处理应用时，可以根据 [State Backend](../state/state_backends.html) 的类型对集群的配置进行优化。
 
-### Heap state backend
+### Heap State Backend
 
-When running a stateless job or using a heap state backend ([MemoryStateBackend](../state/state_backends.html#the-memorystatebackend)
-or [FsStateBackend](../state/state_backends.html#the-fsstatebackend)), set [managed memory](mem_setup.html#托管内存) to zero.
-This will ensure that the maximum amount of memory is allocated for user code on the JVM.
+执行无状态作业或者使用 Heap State Backend（[MemoryStateBackend](../state/state_backends.html#the-memorystatebackend)
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_tuning.zh.md]: Broken link. Should be #memorystatebackend

File: docs/ops/memory/mem_tuning.zh.md [hunk 2]
```diff
@@ -22,66 +22,55 @@ specific language governing permissions and limitations
...
+或 [FsStateBackend](../state/state_backends.html#the-fsstatebackend)）时，建议将[托管内存](mem_setup.html#托管内存)设置为 0。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_tuning.zh.md]: Broken link. Should be #fsstatebackend

File: docs/ops/memory/mem_tuning.zh.md [hunk 3]
```diff
@@ -22,66 +22,55 @@ specific language governing permissions and limitations
...
+### RocksDB State Backend
 
-The [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend) uses native memory. By default,
-RocksDB is setup to limit native memory allocation to the size of the [managed memory](mem_setup.html#托管内存).
-Therefore, it is important to reserve enough *managed memory* for your state use case. If you disable the default RocksDB memory control,
-task executors can be killed in containerized deployments if RocksDB allocates memory above the limit of the requested container size
-(the [total process memory](mem_setup.html#配置总内存)).
-See also [how to tune RocksDB memory](../state/large_state_tuning.html#tuning-rocksdb-memory)
-and [state.backend.rocksdb.memory.managed](../config.html#state-backend-rocksdb-memory-managed).
+[RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend) 使用本地内存。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_tuning.zh.md]: Broken link. Should be #rocksdbstatebackend

File: docs/ops/memory/mem_trouble.zh.md [hunk 4]
```diff
@@ -27,48 +27,43 @@ under the License.
...
 * [`taskmanager.memory.network.max`](../config.html#taskmanager-memory-network-max)
 * [`taskmanager.memory.network.fraction`](../config.html#taskmanager-memory-network-fraction)
 
-## Container Memory Exceeded
+## 容器（Container）内存超用
 
-If a task executor container tries to allocate memory beyond its requested size (Yarn, Mesos or Kubernetes),
-this usually indicates that Flink has not reserved enough native memory. You can observe this either by using an external
-monitoring system or from the error messages when a container gets killed by the deployment environment.
+如果 TaskExecutor 容器尝试分配超过其申请大小的内存（Yarn、Mesos 或 Kubernetes），者通常说明 Flink 没有预留出足够的本地内存。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_trouble.zh.md]: Typo: 者 -> 这

File: docs/ops/memory/mem_trouble.zh.md [hunk 5]
```diff
@@ -27,48 +27,43 @@ under the License.
...
-The exception usually indicates that [JVM metaspace limit](mem_detail.html#jvm-参数) is configured too small.
-You can try to increase the [JVM metaspace option](../config.html#taskmanager-memory-jvm-metaspace-size).
+该异常说明 [JVM Metaspace 限制](mem_detail.html#jvm-参数)过小。
+可以尝试调整 [JVM Metaspace 参数](../config.html#taskmanager-memory-jvm-metaspace-size)。
 
 ## IOException: Insufficient number of network buffers
 
-The exception usually indicates that the size of the configured [network memory](mem_detail.html)
-is not big enough. You can try to increase the *network memory* by adjusting the following options:
+该异常通常说明[网络内存](mem_detail.html)过小。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_trouble.zh.md]: #受限的等比内存部分 could be better.

File: docs/ops/memory/mem_trouble.zh.md [hunk 6]
```diff
@@ -27,48 +27,43 @@ under the License.
...
 
-If a task executor container tries to allocate memory beyond its requested size (Yarn, Mesos or Kubernetes),
-this usually indicates that Flink has not reserved enough native memory. You can observe this either by using an external
-monitoring system or from the error messages when a container gets killed by the deployment environment.
+如果 TaskExecutor 容器尝试分配超过其申请大小的内存（Yarn、Mesos 或 Kubernetes），者通常说明 Flink 没有预留出足够的本地内存。
+可以通过外部监控系统或者容器被部署环境杀掉时的错误信息判断是否存在容器内存超用。
 
-If [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend) is used and the memory controlling is disabled,
-you can try to increase the [managed memory](mem_setup.html#托管内存).
+如果使用了 [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend) 且没有开启内存控制，也可以尝试增大[托管内存](mem_setup.html#托管内存)。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_trouble.zh.md]: Broken link. Should be #rocksdbstatebackend .

File: docs/ops/memory/mem_trouble.zh.md [hunk 7]
```diff
@@ -27,48 +27,43 @@ under the License.
...
 
-Alternatively, you can increase the [JVM overhead](mem_detail.html).
-See also [how to configure memory for containers](mem_tuning.html#容器container的内存配置).
+此外，还可以尝试增大 [JVM 开销](mem_detail.html)。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_trouble.zh.md]: #受限的等比内存部分 could be better.

File: docs/ops/memory/mem_migration.zh.md [hunk 8]
```diff
@@ -22,67 +22,63 @@ specific language governing permissions and limitations
...
         </tr>
     </thead>
     <tbody>
         <tr>
             <td><h5>taskmanager.memory.fraction</h5></td>
             <td>
-                Check the description of the new option <a href="../config.html#taskmanager-memory-managed-fraction)">taskmanager.memory.managed.fraction</a>.
-                The new option has different semantics and the value of the deprecated option usually has to be adjusted.
-                See also <a href="#managed-memory">how to migrate managed memory</a>.
+                请参考新配置参数 <a href="../config.html#taskmanager-memory-managed-fraction)">taskmanager.memory.managed.fraction</a> 的相关描述。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link. Redundant ")".

File: docs/ops/memory/mem_migration.zh.md [hunk 9]
```diff
@@ -121,116 +117,105 @@ The following options are deprecated but if they are still used they will be int
...
+现在，如果仅配置了*Flink总内存*或*进程总内存*，JVM 的堆空间依然是根据总内存减去所有其他非堆内存得到的。
+请参考[如何配置总内存](mem_setup.html#配置总内存)。
 
-Additionally, you can now have more direct control over the JVM heap assigned to the operator tasks
-([`taskmanager.memory.task.heap.size`](../config.html#taskmanager-memory-task-heap-size)),
-see also [Task (Operator) Heap Memory](mem_setup.html#任务算子堆内存).
-The JVM heap memory is also used by the heap state backends ([MemoryStateBackend](../state/state_backends.html#the-memorystatebackend)
-or [FsStateBackend](../state/state_backends.html#the-fsstatebackend)) if it is chosen for streaming jobs.
+此外，你现在可以更直接地控制用于任务和算子的 JVM 的堆内存（[`taskmanager.memory.task.heap.size`](../config.html#taskmanager-memory-task-heap-size)），详见[任务堆内存](mem_setup.html#任务算子堆内存)。
+如果流处理作业选择使用 Heap State Backend（[MemoryStateBackend](../state/state_backends.html#the-memorystatebackend)
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link.

File: docs/ops/memory/mem_migration.zh.md [hunk 10]
```diff
@@ -121,116 +117,105 @@ The following options are deprecated but if they are still used they will be int
...
+或 [FsStateBackend](../state/state_backends.html#the-fsstatebackend)），那么它同样需要使用 JVM 堆内存。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link.

File: docs/ops/memory/mem_migration.zh.md [hunk 11]
```diff
@@ -121,116 +117,105 @@ The following options are deprecated but if they are still used they will be int
...
-### RocksDB state
+### RocksDB State Backend
 
-If the [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend) is chosen for a streaming job,
-its native memory consumption should now be accounted for in [managed memory](mem_setup.html#托管内存).
-The RocksDB memory allocation is limited by the [managed memory](mem_setup.html#托管内存) size.
-This should prevent the killing of containers on [Yarn](../deployment/yarn_setup.html) or [Mesos](../deployment/mesos.html).
-You can disable the RocksDB memory control by setting [state.backend.rocksdb.memory.managed](../config.html#state-backend-rocksdb-memory-managed)
-to `false`. See also [how to migrate container cut-off](#container-cut-off-memory).
+流处理作业如果选择使用 [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend)，它使用的本地内存现在也被归为[托管内存](mem_setup.html#托管内存)。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link.

File: docs/ops/memory/mem_migration.zh.md [hunk 12]
```diff
@@ -121,116 +117,105 @@ The following options are deprecated but if they are still used they will be int
...
+这部分内存将预留给所有未被 Flink 计算在内的内存开销。
+其主要来源是不受 Flink 直接管理的依赖使用的内存，例如 RocksDB、JVM 内部开销等。
+相应的配置参数（`containerized.heap-cutoff-ratio` 和 `containerized.heap-cutoff-min`）现在不再对 TaskExecutor 生效。
+新的内存配置方法引入了新的内存组成部分来具体描述这些内存用量。
 
-In streaming jobs which use [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend), the RocksDB
-native memory consumption should be accounted for as a part of the [managed memory](mem_setup.html#托管内存) now.
-The RocksDB memory allocation is also limited by the configured size of the [managed memory](mem_setup.html#托管内存).
-See also [migrating managed memory](#managed-memory) and [how to configure managed memory now](mem_setup.html#托管内存).
+流处理作业如果使用了 [RocksDBStateBackend](../state/state_backends.html#the-rocksdbstatebackend)，RocksDB 使用的本地内存现在将被归为[托管内存](mem_setup.html#托管内存)。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link.

File: docs/ops/memory/mem_migration.zh.md [hunk 13]
```diff
@@ -121,116 +117,105 @@ The following options are deprecated but if they are still used they will be int
...
+默认情况下，RocksDB 将限制其内存用量不超过[托管内存](mem_setup.html#托管内存)大小。
+请同时参考[如何升级托管内存](托管内存)以及[如何配置托管内存](mem_setup.html#托管内存)。
```
Comment (KarmaGYZ) [inline docs/ops/memory/mem_migration.zh.md]: Broken link. Should be mem_setup.html#托管内存.

File: docs/ops/memory/mem_trouble.zh.md (hunk 5 above)
Comment (xintongsong) [reply docs/ops/memory/mem_trouble.zh.md]: 这里倾向于和英文原文保持一致，不属于原文明显的错误

File: docs/ops/memory/mem_trouble.zh.md (hunk 7 above)
Comment (xintongsong) [reply docs/ops/memory/mem_trouble.zh.md]: 这里倾向于和英文原文保持一致，不属于原文明显的错误

**Comment by xintongsong** [comment]
@KarmaGYZ 
Thanks for the review. Comments addressed.

**Comment by xintongsong** [comment]
Hi @wuchong, would you have time to take a look at this?

**Comment by wuchong** [comment]
Thanks @xintongsong  and @KarmaGYZ  for the work. I will take a look.

**Pull request closed without merging**